src/Illuminate/Validation/Validator.php

```diff
@@ -1200,7 +1200,27 @@ protected function validateEmail($attribute, $value)
      */
     protected function validateUrl($attribute, $value)
     {
-        return filter_var($value, FILTER_VALIDATE_URL) !== false;
+        $pattern = '~^
```

Comment:
we need to attribute where this regex came from - symfony still owns the rights for it